### PR TITLE
Convert to ProFSea python package

### LINKED ISSUE #3: Convert to Python package
Convert the repository into a Python package. This will make it easier to develop and easier to work with in the future.

### PR DESCRIPTION
**Pull request by mo-gregmunday**
This change allows the tool to be installed as a python package, which is of great benefit to developers of the tool - allowing modules to be pulled easily into scripts from all over the codebase with relative imports.

Closes #3.

### CODE REVIEW

File: profsea/config.py
Comment (mo-jenniferweeks) [inline profsea/config.py:1]: Will the new user-settings-greg.yml be named permanently with '-greg'?

**Review by mo-jenniferweeks — CHANGES_REQUESTED** [review]
I approve changes however I have one comment on the name of the user-settings.yml - could you possibly check this is the right file you are uploading?

**Review by mo-MattPalmer — CHANGES_REQUESTED** [review]
All looks good to me, with the exception of profsea/ read_CMIP5_regression_coefficients_sltool_test.html. Could you check whether this file is needed? If yes, please remove/replace username in path.

**Review by mo-jenniferweeks — CHANGES_REQUESTED** [review]
Second comment added regarding profsea/read_CMIP5_regression_coefficients_sltool_test.html file.

File: profsea/read_CMIP5_regression_coefficients_sltool_test.html
[the remark this answers is not in the document]
Comment (mo-gregmunday) [reply profsea/read_CMIP5_regression_coefficients_sltool_test.html:1]: This file is not supposed to be here - I've deleted it now.

File: profsea/config.py
Comment (mo-gregmunday) [reply profsea/config.py:1]: Nope, I've reverted it to the correct one in the latest commit.

**Comment by mo-gregmunday** [comment]
Thanks both @mo-jenniferweeks @mo-MattPalmer, I've made those changes now and have re-requested your review.

**Review by mo-MattPalmer — APPROVED** [review]
All looks good to me - Thanks!

**Review by mo-jenniferweeks — APPROVED** [review]

**Comment by mo-jenniferweeks** [comment]
Merged and deleted branch. Thanks Greg!

**Pull request merged**